example/src/App.tsx

```diff
@@ -101,4 +84,28 @@ const styles = StyleSheet.create({
   },
 });
 
+function useSetupPlayer() {
```

Comment:
@puckey would it be worth wrapping som version of this logic in our JS lib?

Reply:
Originally I implemented retrying on the Android side… but it is probably best left up to the developer on how they want to handle the rejection instead… for example they might decide to want to not retry if a certain amount of time has gone by, or they might want to show a retry button.. or send a notification to the user (although not sure that would be possible from the js side if the app is in the background)